js.go

```diff
@@ -192,15 +192,6 @@ func (nc *Conn) JetStream(opts ...JSOpt) (JetStreamContext, error) {
 	}
 	nc.mu.Unlock()
 
-	if checkAccount {
```

Comment:
If that disappears, then above `jsLastCheck` should disappear too and be removed from the `Conn` object.

Reply:
fixed. JetStream() is now a constructor only. because it can't reallly fail, we can use js.AccountInfo()